### ISSUE TITLE
Translation doesn't work anymore with > 4.3.4

### ISSUE THREAD

**Issue by agitator** [plone/Products.CMFPlone#599]
This worked with Plone 4.3.4, plone.app.multilingual 2.0 and plone.app.contenttypes 1.1b3

When i switch to Plone 4.3.5 the translate action ends up in a "not found" 

```
Cannot locate object at: http://localhost:8081/Plone/de/++addtranslation++463c1998873244a99dbd6243537e3fc4</p>
```

**Comment by saily** [comment]
If it's a `NotFound` the registration of `++addtranslation++` must be wrong. If this is correct this will become a pam issue. I'll check this.

**Comment by frisi** [comment]
a 404 at at_download script has been caused by a new secitury declaration (see #531) so this could be a possible cause too.

**Comment by vangheem** [comment]
Any update? Please close ticket if no longer valid. The plone 4.3.5 release problems have been fixed with 4.3.6.

**Comment by vangheem** [comment]
Closing. Please re-open with further information!
